portfolio/src/main/java/com/google/sps/servlets/TranslationServlet.java

```diff
@@ -25,6 +25,7 @@
 import com.google.sps.utilities.GoogleTranslate;
 import com.google.sps.utilities.FakeTranslate;
 
+/** Servlet that translates a message using the Translate API. */
 @WebServlet("/translate")
 public class TranslationServlet extends HttpServlet {
 
```

Comment:
This comment is for lines 37-43. Please address this in a separate pull request.

A better way to use fakes would be to create an interface. Your method should be unaware of what type of translator it is using (real or fake). Then you can inject the relevant translator depending on the deployment environment.

We can chat about this tomorrow if you are unsure how to do this.

Reply:
I created a new interface that GoogleTranslate and FakeTranslate implement. I'm not exactly sure what you mean by the method should be unaware of what type of translator it is using.

Yes, I think it would be helpful to chat about it!

Reply:
Just made a new PR with these changes!